host/sgx/sgx_enclave_common_wrapper.c

```diff
@@ -113,6 +124,8 @@ static void _load_sgx_enclave_common_impl(void)
     if (_module)
     {
         OE_CHECK(_lookup_function("enclave_create", (void**)&_enclave_create));
+        OE_CHECK(
```

Comment:
Implemented call to enclave_create() when the ex_features are not populated in host/sgx/sgx_enclave_common_wrapper.c

Reply:
As Ming-Wei noted, OE_CHECK would fail if enclave_create_ex is missing. That would imply that if this PR goes in, then OE SDK would only work with newest PSW.

Instead, if enclave_create_ex is not found we can just emit a TRACE message to that effect and continue. Thus module load would succeed, and enclaves that aren't zero-based would still work with only PSWs.

Reply:
Now when _lookup_function() fails for enclave_create_ex(), an OE_TRACE log is writen and the load succeeds.